presto-product-tests/pom.xml

```diff
@@ -47,6 +47,12 @@
         <dependency>
             <groupId>io.prestodb.tempto</groupId>
             <artifactId>tempto-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-lang3</artifactId>
```

Comment:
Why is this needed?

Reply:
commons-lang3 is pulled as a transitive dependency from both Avro (with a higher version) and tempto-core. Initially, I excluded it from Tempto-core to resolve the conflict, but a more simple and reliable solution is to manage it in the dependencyManagement section of the root pom.xml. This ensures consistent versions across the project and prevents issues if Tempto-core updates in the future leads to version mismatch.